### PR TITLE
Chore: Add exceptions to helpers

### LINKED ISSUE #6: Helpers should throw exceptions where applicable
Currently, some do and some don't. I think it's worthwhile to make all helpers that are unfriendly to type-ambiguous arguments throw exceptions when inoperable input is passed in.

### PR DESCRIPTION
**Pull request by erikjung**
Closes #6

This adds exception throwing to the helpers with input value fragility. It also adds new test assertions to make sure these exceptions are thrown when expected.

### CODE REVIEW

File: lib/toFixed.js [hunk 1]
```diff
@@ -12,5 +12,9 @@
  */
 
 module.exports = function toFixed (num) {
-  return num.toFixed(2);
+  var int = parseFloat(num);
```
Comment (tylersticka) [inline lib/toFixed.js]: If we're going to use `parseFloat` (in which case we will accept strings), we should update the associated comment to denote that the parameter may be `{Number|String}` now.

File: test/toFixed.spec.js [hunk 2]
```diff
@@ -10,6 +10,13 @@ tape('toFixed', function (test) {
...
   var actual = template({ number: 1 });
-  test.plan(1);
+  test.plan(2);
   test.equal(actual, expected, 'Works');
+  test.throws(
+    function () {
+      template({ number: 'abc123' })
+    },
+    /number\-like value\.$/,
+    'Errors when passed an unparseable value'
```
Comment (tylersticka) [inline test/toFixed.spec.js]: Interestingly, `parseFloat` _will_ accept the inverse... `123abc` would return `123`. Not a critique, I just learned it this morning and found it interesting!

**Comment by tylersticka** [comment]
Very small nit inline, otherwise LGTM. :+1:

File: lib/toFixed.js (hunk 1 above)
Comment (erikjung) [inline lib/toFixed.js]: Agreed.

File: test/toFixed.spec.js (hunk 2 above)
Comment (erikjung) [inline test/toFixed.spec.js]: Yeah, I saw a similar example in the MDN docs. Perhaps I'll tweak the test to have no numeric characters in that assertion.

**Pull request merged**